src/pages/ImportData.js

```diff
@@ -38,23 +38,26 @@ const styles = {
   }
 }
 
-const ChoiceButton = ({ iconName, title, detail, style, ...props }) => {
+const ChoiceButton = ({ iconName, title, detail, style, onClick, disabled, ...props }) => {
+  const color = disabled ? colors.dark(0.25) : colors.accent(1)
   return h(Clickable, {
     style: {
       ...style,
       padding: '1rem', marginTop: '1rem',
       display: 'flex', alignItems: 'center',
-      border: `1px solid ${colors.accent(1)}`, borderRadius: 4
+      border: `1px solid ${color}`, borderRadius: 4,
+      cursor: disabled ? 'not-allowed' : 'pointer'
     },
-    hover: { backgroundColor: colors.accent(0.1) },
+    hover: disabled ? {} : { backgroundColor: colors.accent(0.1) },
```

Comment:
I think you can probably use `undefined` instead of `{}`, just because I think it makes it clearer that there is no behavior if its disabled.